plugins/ata/fu-ata-device.c

```diff
@@ -394,28 +387,17 @@ fu_ata_device_parse_id (FuAtaDevice *self, const guint8 *buf, gsize sz, GError *
 		fu_device_add_instance_id_full (device, tmp, FU_DEVICE_INSTANCE_FLAG_ONLY_QUIRKS);
 		has_oui_quirk = fu_device_get_vendor (FU_DEVICE (self)) != NULL;
 	}
+	if (self->oui > 0x0) {
+		g_autofree gchar *vendor_id = NULL;
+		vendor_id = g_strdup_printf ("OUI:%06x", self->oui);
+		fu_device_add_vendor_id (device, vendor_id);
+	}
 
 	/* if not already set using the vendor block or a OUI quirk */
 	name = fu_ata_device_get_string (id, 27, 46);
 	if (name != NULL && !has_oui_quirk)
 		fu_ata_device_parse_vendor_name (self, name);
 
-	/* ask user to report data */
-	if (self->oui != 0x0 && !has_oui_quirk && self->unknown_oui_report) {
-		const gchar *url = "https://github.com/fwupd/fwupd/wiki/ATA-Disk:-OUI-Quirk-Required";
```

Comment:
if this is merged, need to remember to update the wiki page about this.

Reply:
Agree; I wasn't sure what your opinions would be.

Reply:
Yeah just want to leave it on the PR as a reminder so if we merge this it's the last open item to do.

Reply:
Done!